LuckyPills/LuckyPills.csproj

```diff
@@ -33,55 +33,55 @@
     </PropertyGroup>
     <ItemGroup>
         <Reference Include="Assembly-CSharp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null">
-          <HintPath>..\..\..\Desktop\Exiled\Assembly-CSharp.dll</HintPath>
+          <HintPath>..\..\..\..\..\..\..\..\Program Files (x86)\Steam\steamapps\common\SCP Secret Laboratory Dedicated Server\SCPSL_Data\Managed\Assembly-CSharp-Publicized.dll</HintPath>
```

Comment:
Seriously? Like it was bad and you didn't make it worse but this needs to be a relative path.

Reply:
lma0